### PR TITLE
Modeling rules for fireeye audit logs

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6502

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/FireEyeHX/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# FireEye HX
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+### Raw syslog audit messages
+In order to configure FireEye HX to send syslog audit logs, refer to FireEye HX "Endpoint Security Server System Administration Guide" (**Configuring a Syslog Server Using the CLI**).
+Make sure to configure the syslog timestamp format to be RFC-3339 UTC.
+
+### CEF format logs
+In order to configure FireEye HX to send cef logs, refer to FireEye HX "Endpoint Security Server System Administration Guide".
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/README.md:10]:
```suggestion
In order to configure FireEye HX to send CEF logs, refer to FireEye HX "Endpoint Security Server System Administration Guide".
```

Maybe we should add a link to the documentation (https://docs.trellix.com/bundle/hx_sag_5-3-0_pdf/resource/HX_SAG_5.3.0_pdf.pdf)

File: Packs/FireEyeHX/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# FireEye HX
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+### Raw syslog audit messages
+In order to configure FireEye HX to send syslog audit logs, refer to FireEye HX "Endpoint Security Server System Administration Guide" (**Configuring a Syslog Server Using the CLI**).
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/README.md:6]: Maybe include a link to the Guide (https://docs.trellix.com/bundle/hx_sag_5-3-0_pdf/resource/HX_SAG_5.3.0_pdf.pdf)

File: Packs/FireEyeHX/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Make sure to configure the syslog timestamp format to be RFC-3339 UTC.
+
+### CEF format logs
+In order to configure FireEye HX to send cef logs, refer to FireEye HX "Endpoint Security Server System Administration Guide".
+
+For further assistant, reach the tech support of FireEye HX.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/README.md:12]:
```suggestion
For further assistant, contact the tech support of FireEye HX.
```

File: Packs/FireEyeHX/ReleaseNotes/2_3_7.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#### Modeling Rules
+##### FireEye HX Modeling Rule
+- Updated the modeling rule for the API logs.
+- Added modeling rule for CEF logs.
+- Added modeling rules for raw syslog logs.
+
+#### Parsing Rules
+##### New: FireEye HX
+- Added parsing rule for API logs.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/ReleaseNotes/2_3_7.md:9]:
```suggestion
Added parsing rule for API logs.
```

**Comment by ShirleyDenkberg** [comment]
@yasta5 Doc review completed. 
FYI: I am not a reviewer and cannot approve these changes.

**Review by evisochek — APPROVED** [review]

**Pull request merged**